README.md

```diff
@@ -114,6 +114,24 @@ Options:
   --help             Show help                                                                                 [boolean]
 ```
 
+### How to deploying a new DAO:
+1. Make sure you have Node.js and NPM installed and updated to a version later that 10.0.0 and 6.0.0 respectively. You can find instructions for installation [here](https://nodejs.org/en/download/), and then verify your version by opening your CMD/ Terminal and type: `node -v` and then `npm -v`.
```

Comment:
later than 15.0.0 ...

Reply:
Not sure what do you mean, the latest is 11.12.0

Reply:
oh...so the recommended is 10.15.3 .
that -> than